azext_iot/digitaltwins/params.py

```diff
@@ -525,3 +525,53 @@ def load_digitaltwins_arguments(self, _):
             options_list=['--yes', '-y'],
             help='Do not prompt for confirmation when assigning required roles.',
         )
+
+    with self.argument_context("dt jobs import") as context:
+        context.argument(
+            "job_id",
+            options_list=["--job-id", "--jid"],
+            help="ID of bulk import job",
+            arg_group="Bulk Import Job",
+        )
+
+        context.argument(
+            "data_file_name",
+            options_list=["--data-file-name", "--dfn"],
+            help="Name of data file input to the bulk import job. The file must be in 'ndjson' format.",
```

Comment:
Unfortunately the API doesn't support this, the API can read the content only from an .ndjson file present in an azure storage container

Reply:
can't we technically create said ndjson for the user?

Reply:
The file needs to be in a user-owned storage account and container (we can't create and maintain these on behalf of the user, since they need to be in user's subscription). Dealing with the permissions of the user-owned storage account in the CLI will be an engineering overhead. Additionally, we will become responsible for parsing the user input to validate the the ndjson file is created correctly. This is the responsibility of the underlying API and hence it is better that user provides the exact file which the API can process (instead of us being intermediaries).

Also the users who don't want to deal with storage accounts/ndjson files have the option to use az model twin and relationship commands to achieve the same goal.